rifServices/src/main/java/org/sahsu/rif/services/graphics/RIFMapsParameters.java

```diff
@@ -291,32 +298,68 @@ public RIFMapsParameter getRIFMapsParameter(String key) {
+		String selectStateText=manager.getColumnFromResultSet(rif40Studies, "select_state", true /* allowNulls */, false /*  allowNoRows */);
+		boolean isDiseaseMappingStudy=true;
+		
+		if (selectStateText != null) {
+			JSONObject selectStateJson = new JSONObject(selectStateText); // Check it parses OK
+			String studyTypeStr = selectStateJson.optString("studyType");
+			if (studyTypeStr != null && studyTypeStr.equals("risk_analysis_study")) {
+				isDiseaseMappingStudy=false;
+				rifLogger.info(getClass(), "rif40_studies.study_id: " + studyID + "; use database select state study type: risk analysis"); 
+				this.studyType = StudyType.RISK_ANALYSIS;
+			}
+			else if (studyType != null && studyType.equals("disease_mapping_study")) {
+				rifLogger.info(getClass(), "rif40_studies.study_id: " + studyID + "; use database select state study type: disease mapping"); 
+			}
+			else {
+				rifLogger.info(getClass(), "rif40_studies.study_id: " + studyID + "; no database select state study type, assume: disease_mapping;" +
+					lineSeparator +
+					"; selectStateJson: " + selectStateJson.toString(2)); 
+			}	
+		}
+		else {
+			rifLogger.info(getClass(), "rif40_studies.study_id: " + studyID + "; no database select state, assume study type: disease_mapping"); 
+		}
 		
-		RIFMapsParameter rifMapsParameter3 = new RIFMapsParameter(	
-			"posterior_probability"		/* resultsColumn */,	
-			"AtlasProbability"			/* Classifier function name */, 
-			"RdYlGn"					/* colorbrewer palette: http://colorbrewer2.org/#type=diverging&scheme=PuOr&n=8 */, 
-			atlasProbabilityBreaks		/* breaks */, 
-			false						/* invert */);		
-		rifMapsParameters.put("diseasemap2", rifMapsParameter3);
+		if (isDiseaseMappingStudy) {
+			RIFMapsParameter rifMapsParameter1 = new RIFMapsParameter(	
+				"relative_risk"		/* resultsColumn */,
+				"quantile"		/* Classifier function name */, 
+				"PuOr"			/* colorbrewer palette: http://colorbrewer2.org/#type=diverging&scheme=PuOr&n=8 */, 
+				9				/* numberOfBreaks */, 
+				true			/* invert */);
+			rifMapsParameters.put("viewermap", rifMapsParameter1);
+					
+			RIFMapsParameter rifMapsParameter2 = new RIFMapsParameter(
+				"smoothed_smr"		/* resultsColumn */,
+				"quantile"		/* Classifier function name */, 
+				"PuOr"			/* colorbrewer palette: http://colorbrewer2.org/#type=diverging&scheme=PuOr&n=8 */, 
+				9				/* numberOfBreaks */, 
+				true			/* invert */);		
+			rifMapsParameters.put("diseasemap1", rifMapsParameter2);
+			
+			RIFMapsParameter rifMapsParameter3 = new RIFMapsParameter(	
+				"posterior_probability"		/* resultsColumn */,	
+				"AtlasProbability"			/* Classifier function name */, 
+				"RdYlGn"					/* colorbrewer palette: http://colorbrewer2.org/#type=diverging&scheme=PuOr&n=8 */, 
+				atlasProbabilityBreaks		/* breaks */, 
+				false						/* invert */);		
+			rifMapsParameters.put("diseasemap2", rifMapsParameter3);
+		}
+		else { // Risk analysis
+			RIFMapsParameter rifMapsParameter1 = new RIFMapsParameter(	
```

Comment:
Parameter1 is the same in both cases, so the if/else doesn't need to include it; just parameters 2 & 3 if it's disease mapping.

Reply:
Fixed